assay-proc-macro/src/lib.rs

```diff
@@ -48,7 +48,27 @@ impl Parse for AssayAttribute {
                 Expr::Lit(ExprLit {
                   lit: Lit::Str(lit_str),
                   ..
-                }) => Some(lit_str.value()),
+                }) => {
+                  let value = lit_str.value();
+                  Some((value, None))
+                },
+                Expr::Tuple(tuple) => {
+                  let mut elements = Vec::new();
+                  for e in tuple.elems.into_iter() {
+                    if let Expr::Lit(ExprLit { lit: Lit::Str(lit_str), ..}) = e {
+                      elements.push(lit_str.value());
+                    } else {
+                      // TODO: Should we return a parsing error to the user here? How?
```

Comment:
Still an open question for me, but not one I think needs to be addressed in this PR! Worth opening an issue on my end.

Reply:
Relevant Issue now -> https://github.com/mgattozzi/assay/issues/13